scripts/bootstrap-aio.yml

```diff
@@ -142,21 +142,27 @@
 
     - name: Add target user vars to user_variables_overrides
       set_fact:
-        user_variables_overrides: "{{ user_variables_overrides|combine(target_user_variables_overrides[TARGET])}}"
+        user_variables_overrides: "{{ user_variables_overrides|combine(target_user_variables_overrides[target])}}"
       when:
-        - TARGET is defined
-        - TARGET in target_user_variables_overrides
+        - target is defined
```

Comment:
Why target and TARGET? (L154 includes both). If you do decide to go with lower case, SCENARIO and TRIGGER should also be lowercased for consitency.